### ISSUE TITLE
Mass Obsoletion: focus on branch 'hematologic disorder' (MONDO:0005570)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6746]
branch: 'hematologic disorder' (MONDO:0005570)

Terms to obsolete: 
obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0015822 | acquired neutropenia
http://purl.obolibrary.org/obo/MONDO_0015818 | aggressive primary cutaneous B-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0015817 | aggressive primary cutaneous T-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0017144 | alpha-thalassemia and related diseases
http://purl.obolibrary.org/obo/MONDO_0016513 | alpha-thalassemia-related diseases
http://purl.obolibrary.org/obo/MONDO_0016599 | autosomal dominant secondary polycythemia
http://purl.obolibrary.org/obo/MONDO_0015679 | autosomal thrombocytopenia with normal platelets
http://purl.obolibrary.org/obo/MONDO_0019138 | bleeding diathesis due to a collagen receptor defect
http://purl.obolibrary.org/obo/MONDO_0017397 | constitutional dyserythropoietic anemia
http://purl.obolibrary.org/obo/MONDO_0020118 | dense granule disease
http://purl.obolibrary.org/obo/MONDO_0018340 | hereditary isolated aplastic anemia
http://purl.obolibrary.org/obo/MONDO_0017057 | hereditary thrombocytopenia with normal platelets
http://purl.obolibrary.org/obo/MONDO_0015975 | hyper-IgM syndrome with susceptibility to opportunistic infections
http://purl.obolibrary.org/obo/MONDO_0015976 | hyper-IgM syndrome without susceptibility to opportunistic infections
http://purl.obolibrary.org/obo/MONDO_0018796 | isolated constitutional thrombocytopenia
http://purl.obolibrary.org/obo/MONDO_0016361 | isolated hereditary giant platelet disorder
http://purl.obolibrary.org/obo/MONDO_0017036 | Langerhans cell histiocytosis in childhood and adulthood
http://purl.obolibrary.org/obo/MONDO_0034872 | large granular lymphocyte leukemia
http://purl.obolibrary.org/obo/MONDO_0020084 | lymphoproliferative disease associated with primary immune disease
http://purl.obolibrary.org/obo/MONDO_0020081 | macrophage or histiocytic tumor
http://purl.obolibrary.org/obo/MONDO_0017146 | sickle cell disease and related diseases
http://purl.obolibrary.org/obo/MONDO_0016667 | sickle cell disease associated with an other hemoglobin anomaly
http://purl.obolibrary.org/obo/MONDO_0019044 | tumor of hematopoietic and lymphoid tissues

**Comment by katiermullen** [comment]
Review spreadsheet here: [https://docs.google.com/spreadsheets/d/1ZEAaoGV1QH83eBy5sw7GFdN7rPRxYiRLtvbTduszq1M/edit#gid=985944731](url)

**Comment by sabrinatoro** [comment]
@katiermullen I have reviewed your spreadsheet and made some comments. This can be pushed to PR whenever you have reviewed (and if you don't have any questions).
Great job.

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1yzM0cfLasL5mVMsEJY5lchngphRQuGqU/edit#gid=244248016 


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab) **!!!!1 term comment will need to be added manually**
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] review "Reviewer-Leave Curator-Stay" Tab [TO DO WILL BE UPDATED ACCORDINGLY]


Note: Once the to-do list is done, all terms will have been reviewed and this issue can be closed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
